A HTLC test case sometimes fails

**Bug Description**

A HTLC test case sometimes fails. E.G. https://github.com/bitshares/bitshares-core/pull/2215/checks?check_run_id=824931271


> /bitshares-core/bitshares-core/tests/tests/htlc_tests.cpp:787: error: in "htlc_tests/htlc_hardfork_test": check good_proposal_id(db).is_authorized_to_execute(db) has failed

The failing check is https://github.com/bitshares/bitshares-core/blob/06b0fe30c25aff4bb931a2e667f90b4ccd6759bf/tests/tests/htlc_tests.cpp#L787

**Impacts**
Describe which portion(s) of BitShares Core may be impacted by this bug. Please tick at least one box.
- [ ] API (the application programming interface)
- [ ] Build (the build process or something prior to compiled code)
- [ ] CLI (the command line wallet)
- [ ] Deployment (the deployment process after building such as Docker, Travis, etc.)
- [ ] DEX (the Decentralized EXchange, market engine, etc.)
- [ ] P2P (the peer-to-peer network for transaction/block propagation)
- [ ] Performance (system or user efficiency, etc.)
- [ ] Protocol (the blockchain logic, consensus, validation, etc.)
- [ ] Security (the security of system or user data, etc.)
- [ ] UX (the User Experience)
- [ ] Other (please add below)

## CORE TEAM TASK LIST
- [ ] Evaluate / Prioritize Bug Report
- [ ] Refine User Stories / Requirements
- [ ] Define Test Cases
- [ ] Design / Develop Solution
- [ ] Perform QA/Testing
- [ ] Update Documentation

Another one, seems related: https://github.com/bitshares/bitshares-core/pull/2334/checks?check_run_id=1752222551
> tests/tests/htlc_tests.cpp:807:error: in "htlc_tests/htlc_hardfork_test": check db.get_global_properties() .parameters.extensions.value.updatable_htlc_options->max_preimage_size == 2048u has failed [19200 != 2048]

--- a/tests/tests/htlc_tests.cpp
+++ b/tests/tests/htlc_tests.cpp
@@ -665,9 +665,7 @@ try {
 BOOST_AUTO_TEST_CASE( htlc_hardfork_test )
 { 
    try {
-      ACTORS( (alice) );
-      int64_t init_balance(10000 * GRAPHENE_BLOCKCHAIN_PRECISION);
-      transfer( committee_account, alice_id, graphene::chain::asset(init_balance) );
+
       {
          // try to set committee parameters before hardfork
          proposal_create_operation cop = proposal_create_operation::committee_proposal(

